tests/plugins/glimpse/test_cms_plugins.py

```diff
@@ -27,6 +27,13 @@ def test_factory_glimpse_title_required():
         """
         GlimpseFactory(title=None)
 
+    @staticmethod
+    def test_factory_glimpse_title_required():
```

Comment:
The function name is not relevant with what your testing and is already defined.

Reply:
@lunika oups. Fixed it.